ClosedXML_Tests/Excel/ImageHandling/PictureTests.cs

```diff
@@ -12,6 +13,7 @@ namespace ClosedXML_Tests
     [TestFixture]
     public class PictureTests
     {
+#if _NETFRAMEWORK_
```

Comment:
It's a matter of personal preference, but I think having a check of the platform inside the test method will indicate more clearly that some parts of the code are not fully cross-platform. I.e.
```
        [Test]
        public void CanAddPictureFromBitmap()
        {
#if !_NETFRAMEWORK_
            Assert.Ignore("Platform is not supported");
#else
            using (var wb = new XLWorkbook())
            ...
```